docs/reference/commandline/service_update.md

```diff
@@ -154,6 +154,47 @@ $ docker service update --mount-rm /somewhere myservice
 myservice
 ```
 
+### Rolling back to the previous version of a service 
+
+Use the `--rollback` options to roll back to the previous version of the service.This will revert the service to the configuration that was in place before the most recent docker service update command.
+
+The following example update the replicated number of service `web` from  4 to 5, and then rollback to previous configuration.
```

Comment:
"The following example updates the number of replicas for the service `web` from 4 to 5, and then rolls back to the previous configuration."